README.md

```diff
@@ -388,3 +388,19 @@ specifically disclaims any liability with respect to this document
 and no contractual obligations are formed either directly or
 indirectly by this document. The technologies, functionality, services,
 and processes described herein are subject to change without notice.
+
+## **Important Notice: This Repository is Deprecated**
+
+**Important Notice**
+
+This repository (fsxa-pattern-library) has been deprecated and is no longer actively maintained. Its usage comes with potential risks and limitations. We recommend users to consider alternative solutions for their needs.
```

Comment:
`Its usage comes with potential risks and limitations. We recommend users to consider alternative solutions for their needs.`
We do not recommend `alternative solutions` - we recommend a specific alternative solution, which is the crownpeak-pwa-template. Please remove these sentences.